utils/systemd/monero-hardened.service

```diff
@@ -0,0 +1,50 @@
+[Unit]
+Description=Monero Full Node
+After=network.target
+
+[Service]
+User=monero
+Group=monero
+WorkingDirectory=/var/lib/monero
+Type=simple
+ExecStart=/usr/local/bin/monerod --config-file /etc/monero/monerod.conf --non-interactive
+Environment="MONERO_RANDOMX_UMASK=8" # Disable RandomX JIT, needed for MemoryDenyWriteExecute=yes
+Environment="MONERO_RANDOMX_FULL_MEM=1" # Use full dataset to speed up verification
+Environment="LD_PRELOAD=/usr/local/lib/libhardened_malloc_default.so" # Protect from memory vulnerabilities, check https://github.com/GrapheneOS/hardened_malloc
```

Comment:
If someone thinks they need this, why wouldn't they install and configure it system wide?